src/web/mjs/connectors/SixMH7.mjs

```diff
@@ -0,0 +1,21 @@
+import SinMH from './templates/SinMH.mjs';
+
+export default class SixMH7 extends SinMH {
+
+    constructor() {
+        super();
+        super.id = '6mh7';
+        super.label = '6漫画 (6mh7)';
+        this.tags = [ 'manga', 'webtoon', 'chinese' ];
+        this.url = 'http://www.6mh7.com';
+
+        this.path = '/sort/1-%PAGE%.html';
+        this.queryManga = 'div.cy_main div.cy_info div.cy_title h1';
+        this.queryMangasPageCount = 'div.NewPages ul li:last-of-type a';
+        this.queryMangas = 'div.cy_list_mh ul li.title a';
+        this.queryChapters = 'ul[id^="mh-chapter-list"] li a';
```

Comment:
this doesn't get all chapters, just the ones that are loaded on page

Reply:
Fixed script to get all chapters